cmd/helm/repo_remove.go

```diff
@@ -41,14 +41,14 @@ func newRepoRemoveCmd(out io.Writer) *cobra.Command {
 	o := &repoRemoveOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "remove [NAME]",
+		Use:     "remove [REPO1 [REPO2 ...]]",
 		Aliases: []string{"rm"},
 		Short:   "remove a chart repository",
```

Comment:
Please update the Short description for this new behavior

Reply:
Got it. :) Will look into it.